audius-cli

```diff
@@ -150,8 +150,9 @@ def auto_upgrade(cron_expression, remove):
         for job in cron.find_comment("audius-cli auto-upgrade"):
             cron.remove(job)
         if not remove:
+            # Pipe auto upgrade logs to auto_upgrade_logs.txt
             job = cron.new(
-                "bash -c 'yes | audius-cli upgrade'",
+                "date >> auto_upgrade_logs.txt; (bash -c 'yes | /usr/local/bin/audius-cli upgrade' 2>&1) >> auto_upgrade_logs.txt; echo >> auto_upgrade_logs.txt",
```

Comment:
this way, we can keep track of what is happening via this cron job + if any issues  come up, we can debug

Reply:
Maybe logging should be optional, perhaps allow path specification.

Reply:
discussed offline -- should be ok to keep as the file should not get too big using the default cadence of upgrading every 6 hours and writes of 1-2kB per pull. (quick math = about 57 weeks for log file to get to 1MB)

Reply:
updated the path to `{manifests_path}/.logs/auto_upgrade.txt` + added `.logs/` to .gitignore